src/index.js

```diff
@@ -62,6 +62,11 @@ function syncReduxAndRouter(history, store) {
       history.pushState(null, routing.path);
     }
   });
+
+  return function unsubscribe() {
```

Comment:
I don't think so. I think just returning a function is fine! Worst case, this should only ever be used once in a project, so it wouldn't be a big deal to do a backwards-incompatible change.